variable.tf

```diff
@@ -48,6 +33,16 @@ variable "tags" {
   type        = map(string)
 }
 
+variable "log_retention_days" {
+  description = "Retention period for CloudWatch logs (in days)"
+  type        = number
+}
+
+variable "parameter_name" {
+  description = "The name of the SSM Parameter that needs to be retrieved"
```

Comment:
What this parameter is for? Let's make that a bit more descriptive

Reply:
It's for the variables used by the pipeline. Sorry for the vague description, I will revise this.